### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by hwbllmnn**
@geostyler/sprinters @geostyler/devs Please review.

### CODE REVIEW

File: src/SldStyleParser.ts [hunk 1]
```diff
@@ -398,7 +401,7 @@ export class SldStyleParser implements StyleParser<string> {
     if (!operator) {
       return;
     }
-    const filter = this.getFilterFromOperatorAndComparison(operator, sldFilter);
+    const filter = this.getFilterFromOperatorAndComparison(operator as keyof typeof COMPARISON_MAP, sldFilter);
```
Comment (KaiVolland) [inline src/SldStyleParser.ts:404]: Any chance to simplify `operator as keyof typeof COMPARISON_MAP`?
Maybe u can type the `SldOperator`s and then use `Record<SldOperator, Operator>` or something similiar.

File: src/SldStyleParser.ts [hunk 2]
```diff
@@ -505,26 +511,27 @@ export class SldStyleParser implements StyleParser<string> {
...
+        return this.getFilterFromOperatorAndComparison(operatorName as any, op);
       });
       filter = [
         combinationOperator,
         ...filters
       ];
-    } else if (Object.keys(SldStyleParser.negationOperatorMap).includes(sldOperatorName)) {
-      const negationOperator = SldStyleParser.negationOperatorMap[sldOperatorName];
+    } else if (Object.keys(NEGATION_OPERATOR_MAP).includes(sldOperatorName)) {
+      const negationOperator = NEGATION_OPERATOR_MAP[sldOperatorName as keyof typeof NEGATION_OPERATOR_MAP];
```
Comment (KaiVolland) [inline src/SldStyleParser.ts:526]: see above

File: src/SldStyleParser.ts [hunk 3]
```diff
@@ -505,26 +511,27 @@ export class SldStyleParser implements StyleParser<string> {
...
       const negatedOperator = Object.keys(sldFilter[sldOperatorName][0])[0];
       const negatedComparison = sldFilter[sldOperatorName][0];
       const negatedFilter: GsFilter = this.getFilterFromOperatorAndComparison(
-        negatedOperator,
+        negatedOperator as any,
```
Comment (KaiVolland) [inline src/SldStyleParser.ts:530]:
```suggestion
        negatedOperator as Operator,
```

File: src/SldStyleParser.ts [hunk 4]
```diff
@@ -1755,25 +1759,26 @@ export class SldStyleParser implements StyleParser<string> {
     };
 
     const fontCssParameters: any[] = Object.keys(textSymbolizer)
-      .filter((property: any) => property !== 'kind' && fontPropertyMap[property])
+      .filter((property: any) => property !== 'kind' && fontPropertyMap[property as keyof typeof fontPropertyMap])
```
Comment (KaiVolland) [inline src/SldStyleParser.ts:1762]: see above and every following apperance of `as keyof typeof`

File: src/SldStyleParser.ts (hunk 1 above)
Comment (hwbllmnn) [reply src/SldStyleParser.ts]: Extracted it out into a type.

File: src/SldStyleParser.ts (hunk 2 above)
Comment (hwbllmnn) [reply src/SldStyleParser.ts]: See above

File: src/SldStyleParser.ts (hunk 3 above)
Comment (hwbllmnn) [reply src/SldStyleParser.ts:534]: That doesn't typecheck unfortunately

**Review by KaiVolland — APPROVED** [review]

**Pull request merged**